api/routes/validation/orderValidation.js

```diff
@@ -0,0 +1,69 @@
+const {
+    RequestValidationError,
+    ArgumentIsIncorrectType,
+    UpdateError
+} = require('@helpers/errors');
+
+class OrderValidation {
+    static Validate(request, response, next) {
+        try {
+            switch (request.method) {
+                case "POST": 
+                    OrderValidation.ValidateCreate(request.body);
+                    break;
+                case "GET":
+                    OrderValidation.ValidateIdIsNaN(request.params.customerId);
```

Comment:
I guess you wanted to pass here `request.params.orderId` instead. Although we will have to implement `getOrdersForCustomer` so you might want to validate request for that too